recipes-ni/sysconfig-settings/sysconfig-settings_1.0.bb

```diff
@@ -127,6 +129,12 @@ do_install () {
 	chown ${LVRT_USER}:${LVRT_GROUP} ${D}${settingsdatadir}/
 	install -m 0644 ${S}/systemsettings/* ${D}${settingsdatadir}/
 
+	# Create shared systemsettingsdir with appropriate permissions and ownership
+	install -d -m 0775 ${D}/${systemsettingsbasedir}
+	chown ${LVRT_USER}:${LVRT_GROUP} ${D}/${systemsettingsbasedir}
```

Comment:
Since `/var/local/natinst` isn't owned by this package, I'm not sure we care to set the ownership here. Presumably, that directory is owned by [`base-files-nilrt`](https://github.com/ni/meta-nilrt/blob/nilrt/master/hardknott/recipes-ni/base-files-nilrt/base-files-nilrt_1.0.bb#L28), which this package should depend upon. So it should already exist and have whatever permissions are defined in `base-files-nilrt` when this IPK is unpacked.

Reply:
Fair enough, I added the additional DEPEND and changed the recipe to use the -o and -g flags instead of chown-ing the directory after the fact.

Reply:
I don't think you need the `-o` or `-g` at all, but this is fine.